sigstore/_cli.py

```diff
@@ -713,9 +713,7 @@ def _sign_common(
             else:
                 sig_output = sys.stdout
 
-            signature = base64.b64encode(
-                result._inner.message_signature.signature
-            ).decode()
```

Comment:
NB: This was previously just wrong for DSSE outputs, but we didn't notice since nothing really uses the `--signature` flag anymore.